alacritty_terminal/src/term/mod.rs

```diff
@@ -1586,7 +1591,30 @@ impl<T: EventListener> Handler for Term<T> {
 
     #[inline]
     fn move_forward_tabs(&mut self, count: u16) {
-        trace!("[unimplemented] Moving forward {} tabs", count);
+        trace!("Moving forward {} tabs", count);
+        self.damage_cursor();
+
+        let num_cols = self.columns();
+        let old_col = self.grid.cursor.point.column.0;
+        for _ in 0..count {
+            let mut col = self.grid.cursor.point.column;
+
+            if col == num_cols - 1 {
```

Comment:
Grid has a `last_column` method, which returns a `Column` rather than a `usize`. It doesn't look like it would clean anything up here but I just wanted to put this as a reminder.

Reply:
We don't have step/etc defined for `Column`, thus it'll make it worse. Like the way it's right now is fine due to implicit type changes.